### ISSUE TITLE
Transfer data from backend to frontend

### ISSUE THREAD

**Issue by ernivani** [KBVE/cloud-function-lolpython#7]
Hey i'm actually wondering how you transfer data from a form like we ask for a word and generate an image in the python and send it into the frontend

**Comment by h0lybyte** [comment]
Like PostMan? A way to send data to the backend and get data back? 

OR The actual front end of the NodePy?

**Comment by ernivani** [comment]
yes like have a form in html js and with a post send databack to the front
